src/components/WalletInit/RestoreWallet/VerifyRestoredWallet.js

```diff
@@ -0,0 +1,156 @@
+// @flow
+
+import React from 'react'
+import {compose} from 'redux'
+import {connect} from 'react-redux'
+import {View, Image, FlatList} from 'react-native'
+import {withHandlers, withStateHandlers} from 'recompose'
+import {SafeAreaView, withNavigation} from 'react-navigation'
+import {injectIntl, defineMessages, intlShape} from 'react-intl'
+
+import {Text, Button, StatusBar, BulletPointItem} from '../../UiKit'
+import {WALLET_INIT_ROUTES} from '../../../RoutesList'
+
+import {withNavigationTitle} from '../../../utils/renderUtils'
+
+import styles from './styles/VerifyRestoredWallet.style'
+
+import type {ComponentType} from 'react'
+import type {Navigation} from '../../../types/navigation'
+import WalletAddress from './WalletAddress'
+
+import mockAvatar from '../../../assets/mockAssets/avatar.png'
+
+const messages = defineMessages({
+  title: {
+    id: 'components.walletinit.verifyrestoredwallet.title',
+    defaultMessage: '!!!Verify restored wallet',
+    description: 'some desc',
+  },
+  checksumLabel: {
+    id: 'components.walletinit.verifyrestoredwallet.checksumLabel',
+    defaultMessage: '!!!Chacksum label',
+    description: 'some desc',
+  },
+  instructionLabel: {
+    id: 'components.walletinit.verifyrestoredwallet.instructionLabel',
+    defaultMessage: '!!!Be careful about wallet restoration:',
+    description: 'some desc',
+  },
+  instructions1: {
+    id: 'components.walletinit.verifyrestoredwallet.instructionLabel-1',
+    defaultMessage: '!!!Wallet Instruction',
+    description: 'some desc',
+  },
+  instructions2: {
+    id: 'components.walletinit.verifyrestoredwallet.instructionLabel-2',
+    defaultMessage: '!!!Wallet Instruction',
+    description: 'some desc',
+  },
+  instructions3: {
+    id: 'components.walletinit.verifyrestoredwallet.instructionLabel-3',
+    defaultMessage: '!!!Wallet Instruction',
+    description: 'some desc',
+  },
+  walletAddressLabel: {
+    id: 'components.walletinit.verifyrestoredwallet.walletAddressLabel',
+    defaultMessage: '!!!Wallet Instruction',
+    description: 'some desc',
+  },
+  buttonText: {
+    id: 'components.walletinit.verifyrestoredwallet.buttonText',
+    defaultMessage: '!!!Wallet Instruction',
+    description: 'some desc',
+  },
+})
+
+const walletRestaurationIntruction = (formatMessage) => [
+  formatMessage(messages.instructions1),
+  formatMessage(messages.instructions2),
+  formatMessage(messages.instructions3),
+]
+
+const walletAddresses = [
+  'DdfFfGgXxCcXcV … hHhJKKHgfdaa',
```

Comment:
Probably this shouldn't be hard-coded

Reply:
it's ok, he only implemented the UI for now